docs/development.md

```diff
@@ -176,3 +177,167 @@ $ yarn run [name_command]
+
+### 7.1 API
+
+The AB test framework has the following methods:
+
+#### `init()`
+This function initialize the AB test framework by executing the following steps:
+1. Read the MVT (multi variant) id from the cookie of the user. If the user has no mvt is his or her browser, 
+it generate a new MVTid and save it in the cookie.
+
+2. From the MVT, it generate the `Participation` object. The steps to build that object are:
+   * Read the participation from `localStorage`
+   * Check if the test is active.
+   * if the test is not in `localStorage`, it assign one variant from the `mvtId`.
+   * Finally it override the variant if the user pass the variant in the url in the way of: `#ab-[name_of_test]=[variant_name]`.
+
+3. Save the `Participation` object in `localStorage`. 
+     
+
+#### `getVariantsAsString(participation: Participations)`
+It receive the participation object and returns everything as string in the following format:
+
+`test1=variantA; test2=variantB`
+
+
+#### `trackOphan( testId: TestId, variant: string, complete?: boolean = false, campaignCodes?: string[] = [])`
+Track event using `tracker-js` from ophan. 
+
+
+#### `abTestReducer(state: Participations = {}, action: Action)`
+Reducer function to be included in the reducer of the page that will have import the AB test framework.
+
+### 7.2 Set up the AB test framework in a new page of the site
+
+#### Step 1: Initialize the AB test framework on the page you are working on
+In order to use the AB test framework you have to initialize it in your page. Therefore, you have to call the 
+`init` function of the module. That called will return a `Participation` object and you have to set that object in the 
+Redux store.
+
+
+```javascript 1.8
+
++// ----- AB Tests ----- //
+ +
+ +const participation = abTest.init();
+  
+ 
+ // ----- Redux Store ----- //
+  
+  const store = createStore(reducer);
+  
+ +store.dispatch({ type: 'SET_AB_TEST_PARTICIPATION', payload: participation });
+ 
+```
+ 
+ A real example of this you can find it [here](https://github.com/guardian/support-frontend/pull/67/files#diff-bdf2dc8b3411cc1e5f83ca22c698e7b3R41).
+
+#### Step 2: Add the AbTest reducer to your store
+
+In order to be able to understand Redux AB actions, you have to add the `abTestReducer` to your page's reducer as follows:
+ 
+```javascript 1.8
+export default combineReducers({
+  contribution,
+  intCmp,
+  abTests,
+});
+```
+  
+You can find a real example of this [here](https://github.com/guardian/support-frontend/pull/67/files#diff-c1f0bb180b22e8e0da8bde14c6b411c4R122).
+
+
+### 7.3 Implementation of a test
+
+#### Step 0: Define your experiment 
+First of all you have to **design the experiment** that you want to run. The experiment consist of a hypothesis and n 
+amount of variants. For example:
+   
+> Hypothesis: By changing [`element_to_test`], it will `[increase|decrease]` the `[put_some_metric]` by **`number`%**
```

Comment:
It might be easier if you made this an example rather than completely generic.

By changing the background colour to black, it will increase the conversion rate by 5%